### PR TITLE
Complete homework #3

### PR DESCRIPTION
**Pull request by AnastasiiaNashchokina**

### CODE REVIEW

**Comment by AntBelyi** [comment]
remove it please 
Screenshot_10

**Comment by AntBelyi** [comment]
Do we really need it here?
Screenshot_6

File: src/main/java/hw_3/Task1.java [hunk 1]
```diff
@@ -0,0 +1,33 @@
+package hw_3;
+
+import java.util.Arrays;
+import java.util.StringJoiner;
+
+import static java.awt.SystemColor.text;
```
Comment (AntBelyi) [inline src/main/java/hw_3/Task1.java]: These unused imports should be removed
Screenshot_11

File: src/main/java/hw_3/Task1.java [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+public class Task1 {
+    public static void main(String[] args) {
+        // ДЗ рядки
+        //Виведіть у консоль один рядок з числами з масиву (які ми створювали в 1-му завданні), розділеними “;”
```
Comment (AntBelyi) [inline src/main/java/hw_3/Task1.java]: There should be no Latin words

File: src/main/java/hw_3/Task1.java [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+        StringBuilder result = new StringBuilder();
+        for (int i = 0; i < array.length; i++) {
+            result.append(array[i]);
+            if (i < array.length - 1) {
+                result.append(";");
+            }
+        }
+        System.out.println(result.toString());
+        // Створіть змінну із рядком “Мама мила раму”.
+        //Створіть масив зі всіх слів, які зустрічаються у цьому рядку.
```
Comment (AntBelyi) [inline src/main/java/hw_3/Task1.java]: There should be no Latin words

File: src/main/java/hw_3/Task1.java [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+        String message = "Mother washed the frame";
+        String[] words = message.split(" ");
+        for (String word : words) {
+            System.out.println(word);
+        }
+        /*Створіть рядок "Я хочу перенесення рядків."
+        Замініть всі символи пробілу в рядку на символи перенесення рядків та виведіть у консоль.*/
```
Comment (AntBelyi) [inline src/main/java/hw_3/Task1.java]: There should be no Latin words

File: src/main/java/hw_3/Task2.java [hunk 5]
```diff
@@ -0,0 +1,40 @@
+package hw_3;
+
+import java.util.Arrays;
+
+public class Task2 {
+    public static void main(String[] args) {
+        //ДЗ масиви
+        //Створіть масив із цілих чисел і помістіть відразу 10 значень. Пройдіться ними і змініть значення.
```
Comment (AntBelyi) [inline src/main/java/hw_3/Task2.java]: There should be no Latin words

File: src/main/java/hw_3/Task2.java [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+        array[2] = 3;
+        array[3] = 4;
+        array[4] = 5;
+        array[5] = 6;
+        array[6] = 7;
+        array[7] = 8;
+        array[8] = 9;
+        array[9] =10;
+        System.out.println(Arrays.toString(array));
+        //Створіть масив із дробових чисел (5 елементів)
```
Comment (AntBelyi) [inline src/main/java/hw_3/Task2.java]: There should be no Latin words

File: src/main/java/hw_3/Task2.java [hunk 7]
```diff
@@ -0,0 +1,40 @@
...
+        double[] array_1 = new double[5];
+        System.out.println(Arrays.toString(array_1));
+        //Створіть 3 змінних таких типів: byte, int, long. Створіть масив типу int. Спробуйте туди покласти змінні.
```
Comment (AntBelyi) [inline src/main/java/hw_3/Task2.java]: There should be no Latin words

File: src/main/java/hw_3/Task2.java [hunk 8]
```diff
@@ -0,0 +1,40 @@
...
+        byte a = 43;
+        int b = 13;
+        long c = 123;
+        int[] array_2 = new int[3];
+        array_2[0] = a;
+        array_2[1] = b;
+        array_2[2] = (int) c;
+        System.out.println(Arrays.toString(array_2));
+        //Створіть будь-який двовимірний масив, заповніть будь-якими даними та виведіть у консоль
```
Comment (AntBelyi) [inline src/main/java/hw_3/Task2.java]: There should be no Latin words

File: src/main/java/hw_3/Task3.java [hunk 9]
```diff
@@ -0,0 +1,22 @@
+package hw_3;
+
+public class Task3 {
+    public static void main(String[] args) {
+        // ДЗ Масиви та рядки
+        /*Створіть рядок "Hello world!"
```
Comment (AntBelyi) [inline src/main/java/hw_3/Task3.java]: There should be no Latin words

File: src/main/java/hw_3/Task3.java [hunk 10]
```diff
@@ -0,0 +1,22 @@
...
+        - Виведіть перший символ
+        - Виведіть останній символ
+        - Створіть масив символів, покладіть туди усі символи слова world
+        - Вивести масив у консоль
+        - Вивести в консоль передостанній елемент масиву, не використовуючи прямий доступ за індексом (arr[3] – не згодиться)
+        */
```
Comment (AntBelyi) [inline src/main/java/hw_3/Task3.java:12]: There should be no Latin words

File: src/main/java/hw_3/Task1.java [hunk 11]
```diff
@@ -0,0 +1,33 @@
...
+        //Виведіть у консоль один рядок з числами з масиву (які ми створювали в 1-му завданні), розділеними “;”
+        int[] array = {1, 2, 3, 4, 5};
+        StringBuilder result = new StringBuilder();
+        for (int i = 0; i < array.length; i++) {
+            result.append(array[i]);
+            if (i < array.length - 1) {
+                result.append(";");
+            }
+        }
+        System.out.println(result.toString());
```
Comment (AntBelyi) [inline src/main/java/hw_3/Task1.java]: You can use just 'result' in System.out.println. without .toString()

File: src/main/java/hw_3/Task1.java [hunk 12]
```diff
@@ -0,0 +1,33 @@
...
+        //Створіть масив зі всіх слів, які зустрічаються у цьому рядку.
+        String message = "Mother washed the frame";
+        String[] words = message.split(" ");
+        for (String word : words) {
+            System.out.println(word);
+        }
+        /*Створіть рядок "Я хочу перенесення рядків."
+        Замініть всі символи пробілу в рядку на символи перенесення рядків та виведіть у консоль.*/
+        String str = "I want line breaks";
+        String replacedstr = str.replace(" ", "/");
```
Comment (AntBelyi) [inline src/main/java/hw_3/Task1.java]: "/" is the wrong symbol. You should to use
Screenshot_12

So, all words in console should be from the new line

File: src/main/java/hw_3/Task2.java [hunk 13]
```diff
@@ -0,0 +1,40 @@
...
+        array[3] = 4;
+        array[4] = 5;
+        array[5] = 6;
+        array[6] = 7;
+        array[7] = 8;
+        array[8] = 9;
+        array[9] =10;
+        System.out.println(Arrays.toString(array));
+        //Створіть масив із дробових чисел (5 елементів)
+        double[] array_1 = new double[5];
```
Comment (AntBelyi) [inline src/main/java/hw_3/Task2.java]: Use more specific name for array. array_1 s a bad name

File: src/main/java/hw_3/Task3.java [hunk 14]
```diff
@@ -0,0 +1,22 @@
...
+        String hello = "Hello World!";
+        char first = hello.charAt(0);
+        char last = hello.charAt(hello.length() - 1);
+        System.out.println(first);
+        System.out.println(last);
+        char[] world = {'w','o','r','l','d'};
+        System.out.println(world);
+        System.out.println(world[world.length-2]);
```
Comment (AntBelyi) [inline src/main/java/hw_3/Task3.java]: Please ad extra lines before and after '-'

File: src/main/java/hw_3/Task3.java [hunk 15]
```diff
@@ -0,0 +1,22 @@
...
+        - Створіть масив символів, покладіть туди усі символи слова world
+        - Вивести масив у консоль
+        - Вивести в консоль передостанній елемент масиву, не використовуючи прямий доступ за індексом (arr[3] – не згодиться)
+        */
+        String hello = "Hello World!";
+        char first = hello.charAt(0);
+        char last = hello.charAt(hello.length() - 1);
+        System.out.println(first);
+        System.out.println(last);
+        char[] world = {'w','o','r','l','d'};
```
Comment (AntBelyi) [inline src/main/java/hw_3/Task3.java]: You need do get a 'world' word from the "Hello world!" string and then you need to put all symbols from 'world' word into array.
For now, you just create an array and put there symbols. You need to practise string methods

File: src/main/java/hw_3/Task3.java (hunk 10 above)
Comment (AnastasiiaNashchokina) [reply src/main/java/hw_3/Task3.java:12]: fixed

File: src/main/java/hw_3/Task3.java (hunk 9 above)
Comment (AnastasiiaNashchokina) [reply src/main/java/hw_3/Task3.java]: I wrote it for myself! I could have just deleted it, but okay - I'll translate it. Fixed

File: src/main/java/hw_3/Task3.java (hunk 14 above)
Comment (AnastasiiaNashchokina) [reply src/main/java/hw_3/Task3.java]: fixed

File: src/main/java/hw_3/Task1.java (hunk 1 above)
Comment (AnastasiiaNashchokina) [reply src/main/java/hw_3/Task1.java]: fixed

File: src/main/java/hw_3/Task1.java (hunk 2 above)
Comment (AnastasiiaNashchokina) [reply src/main/java/hw_3/Task1.java]: fixed

File: src/main/java/hw_3/Task1.java (hunk 11 above)
Comment (AnastasiiaNashchokina) [reply src/main/java/hw_3/Task1.java]: fixed

File: src/main/java/hw_3/Task1.java (hunk 3 above)
Comment (AnastasiiaNashchokina) [reply src/main/java/hw_3/Task1.java]: fixed

File: src/main/java/hw_3/Task1.java (hunk 12 above)
Comment (AnastasiiaNashchokina) [reply src/main/java/hw_3/Task1.java]: fixed

File: src/main/java/hw_3/Task1.java (hunk 4 above)
Comment (AnastasiiaNashchokina) [reply src/main/java/hw_3/Task1.java]: fixed

File: src/main/java/hw_3/Task2.java (hunk 5 above)
Comment (AnastasiiaNashchokina) [reply src/main/java/hw_3/Task2.java]: fixed

File: src/main/java/hw_3/Task2.java (hunk 6 above)
Comment (AnastasiiaNashchokina) [reply src/main/java/hw_3/Task2.java]: fixed

File: src/main/java/hw_3/Task2.java (hunk 13 above)
Comment (AnastasiiaNashchokina) [reply src/main/java/hw_3/Task2.java]: "numbers" is ok?

File: src/main/java/hw_3/Task2.java (hunk 7 above)
Comment (AnastasiiaNashchokina) [reply src/main/java/hw_3/Task2.java]: fixed

File: src/main/java/hw_3/Task2.java (hunk 8 above)
Comment (AnastasiiaNashchokina) [reply src/main/java/hw_3/Task2.java]: fixed

File: src/main/java/hw_3/Task1.java (hunk 1 above)
Comment (AntBelyi) [reply src/main/java/hw_3/Task1.java]: not

File: src/main/java/hw_3/Task1.java (hunk 2 above)
Comment (AntBelyi) [reply src/main/java/hw_3/Task1.java]: not

File: src/main/java/hw_3/Task3.java (hunk 14 above)
Comment (AntBelyi) [reply src/main/java/hw_3/Task3.java]: not

File: src/main/java/hw_3/Task2.java (hunk 13 above)
Comment (AntBelyi) [reply src/main/java/hw_3/Task2.java]: yes

File: src/main/java/hw_3/Task1.java (hunk 12 above)
Comment (AntBelyi) [reply src/main/java/hw_3/Task1.java]: not

File: src/main/java/hw_3/Task1.java (hunk 11 above)
Comment (AntBelyi) [reply src/main/java/hw_3/Task1.java]: not

File: src/main/java/hw_3/Task2.java (hunk 5 above)
Comment (AntBelyi) [reply src/main/java/hw_3/Task2.java]: not

File: src/main/java/hw_3/Task2.java (hunk 6 above)
Comment (AntBelyi) [reply src/main/java/hw_3/Task2.java]: not

File: src/main/java/hw_3/Task2.java (hunk 7 above)
Comment (AntBelyi) [reply src/main/java/hw_3/Task2.java]: not

File: src/main/java/hw_3/Task3.java (hunk 9 above)
Comment (AntBelyi) [reply src/main/java/hw_3/Task3.java]: not

File: src/main/java/hw_3/Task3.java (hunk 10 above)
Comment (AntBelyi) [reply src/main/java/hw_3/Task3.java:12]: not

File: src/main/java/hw_3/Task3.java (hunk 15 above)
Comment (AnastasiiaNashchokina) [reply src/main/java/hw_3/Task3.java]: fixed

**Comment by AnastasiiaNashchokina** [comment]
> Do we really need it here? Screenshot_6

fixed

**Comment by AnastasiiaNashchokina** [comment]
> Do we really need it here? Screenshot_6

fixed

**Comment by AnastasiiaNashchokina** [comment]
> remove it please Screenshot_10

fixed

**Comment by AnastasiiaNashchokina** [comment]
> Do we really need it here? Screenshot_6

fixed

**Comment by AnastasiiaNashchokina** [comment]
> remove it please Screenshot_10

fixed

**Comment by AnastasiiaNashchokina** [comment]
> Do we really need it here? Screenshot_6

fixed

**Pull request merged**